README.md

```diff
@@ -42,7 +42,9 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
 ```js
-// write code here
+for (var i = 0; i <= foods.length; i++) {
+   console.log("I want to eat a " + foods[0][i]);
+ }
```

Comment:
:+1:
Close.  Try adding another fruit to the foods array.  Does it list ALL the fruits?  Why not?